workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/ScreenViewHolder.kt

```diff
@@ -26,6 +26,16 @@ public interface ScreenViewHolder<in ScreenT : Screen> {
    * maintained correctly, and [showing] keeps working.
    */
   public val runner: ScreenViewRunner<ScreenT>
+
+  public companion object {
```

Comment:
Call sites aren't changed by this, it still looks like a constructor invocation of the `ScreenViewHolder` interface. Just seems more discoverable to me.